src/clj_http/conn_mgr.clj

```diff
@@ -152,16 +143,12 @@
                    (get-keystore-scheme-registry config)
 
                    :else regular-scheme-registry)]
-    (PoolingClientConnectionManager.
-     registry timeout java.util.concurrent.TimeUnit/SECONDS)))
-
-(def dmcpr ConnPerRouteBean/DEFAULT_MAX_CONNECTIONS_PER_ROUTE)
+    (PoolingHttpClientConnectionManager. registry)))
```

Comment:
I am not sure how to create a PoolingHttpClientConnectionManager with a timetolive. The constructor is a bit cryptic to me :) http://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html

Maybe we should use http://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/client/HttpClientBuilder.html#setConnectionTimeToLive(long,%20java.util.concurrent.TimeUnit) instead?